Makefile

```diff
@@ -43,15 +43,15 @@ CONTROLLER_IMAGE_FILE  := dist/controller-image.marker
 
 # perform static compilation
 STATIC_BUILD          ?= true
-CI                    ?= false
+STATIC_FILES          ?= false
```

Comment:
I this this should be `true` by default

Reply:
but only `false` triggers the following for the targets `ui/dist/node_modules.marker` and `ui/dist/app/index.html`

```
ifeq ($(STATIC_FILES),false)
	yarn --cwd ui build
else
	echo "Built without static files" > ui/dist/app/index.html
endif
```

Reply:
You would need to flip that condition too. Maybe others.

Reply:
@alexec I flipped all conditions